ui/app/AppLayouts/TradingCenter/controls/TradingCenterTokenDelegate.qml

```diff
@@ -0,0 +1,196 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+
+StatusItemDelegate {
+    id: root
+
+    /** Input property representing token name **/
+    property string tokenName
+    /** Input property representing token symbol **/
+    property string tokenSymbol
+    /** Input property representing icon source for token **/
+    property string iconSource
+    /** Input property representing token market price **/
+    property string price
+    /* Input property representing color of text -
+    - green: if change is positive
+    - red: if change is negative */
+    property string changePct24HourColor
+    /* Input property representing percentage change in
+    market price in the last 24 hours */
+    property string changePct24Hour
+    /* Input property representing volume
+    in last 24 hours */
+    property string volume24Hour
+    /* Input property representing token market cap */
+    property string marketCap
+    /* Input property if token is loading */
+    property bool loading
+
+    QtObject {
+        id: d
+        // Split into 4 different columns of equal width
+        readonly property int columnWidth:
+            (root.width - indexText.width - icon.width - nameSymbolBox.width - 28) / 4
+    }
+
+    implicitHeight: 76
+    padding: 0
+    cursorShape: Qt.ArrowCursor
```

Comment:
Intended?

Reply:
Yes, we want to show a hover start but no interaction

Reply:
OK